packages/portalnetwork/src/wire/utp/Socket/ReadSocket.ts

```diff
@@ -42,10 +42,7 @@ export class ReadSocket extends UtpSocket {
       expected = this.ackNr + 1 === packet.header.seqNr
     }
     this.setSeqNr(this.getSeqNr() + 1)
-    if (!this.reader) {
-      this.reader = new ContentReader(packet.header.seqNr)
-      this.reader.bytesExpected = Infinity
-    }
+
```

Comment:
Are we confident there's never a place where a reader won't have been instantiated?

Reply:
I moved this line into the request class, since it is aware of the version and the socket is not